server/views/components/summary-card/goal-summary-card.njk

```diff
@@ -35,11 +35,14 @@
             }
         }) %}
         <div class="goal-summary-card__info">
-            {% if params.goal.targetDate and params.goal.status != "ACHIEVED" %}
+            {% if params.goal.targetDate and params.goal.status != "ACHIEVED" and params.goal.status != "REMOVED" %}
                 <p>{{ locale.info.aimToAchieveDate }}</p>
             {% endif %}
+            {% if params.goal.status == "REMOVED" %}
+                <p>{{ locale.info.removedDate }}</p>
+            {% endif %}
```

Comment:
instead of consecutive `if`s which test the same parameter value you can use `elif`: https://mozilla.github.io/nunjucks/templating.html#if